docs/geedocs/5.2.3/answer/6051700.html

```diff
@@ -126,6 +126,27 @@ <h3>Symbolic links</h3>
   <li><strong>soft-link</strong>: the disconnected database folder and published_dbs folder are on different logical volumes.</li>
   <li>If the disconnected database folder is in <code>/tmp</code>, the directory files will be copied. </li>
 </ul>
+<h3>Rewrite KML Layer URLs</h3>
+<p>If the disconnected database contains KML Layers, the KML Layer's URLs may require adjustments in order to resolve the KML content in a disconnected environment.   (See <a href="../answer/2760457.html">Add KML Layers </a>). Use the <code>gerewritedbroot</code> command to update the disconnected database KML Links with new hostname, ports paths or filenames. </p>
+
+<li>Identify which dbroot file needs to be updated, and store in an environmnt variable for later use:
+  <p><code>export TARGET_DBROOT='/media/ddb/test_ver001/gevol/assets/Databases/test.kdatabase/gedb.kda/ver001/gedb/targets/new_target/dbroots/dbroot.v5p.DEFAULT'</code></p></li>
+<li>Rewrite the KML Layer Links in the database on the Server. 
+    <p><code>/opt/google/bin/gerewritedbroot --source http://your_stream_server/new_target \</code></p>
```

Comment:
Other `<code>` elements on this page use line break elements for multi-line code.

I think `<code>` elements should render white space as pre-formatted, but that may require more extensive changes.

I think you should change this to use one `<code>` element.